README.md

```diff
@@ -7,35 +7,32 @@
 ![Python 3.12](https://img.shields.io/badge/Python-3.12-blue.svg)
 [![Docker Image Version (latest semver)](https://img.shields.io/docker/v/safeglobal/safe-cli?label=Docker&sort=semver)](https://hub.docker.com/r/safeglobal/safe-cli)
 
-# Safe-CLI
+# Safe CLI
 
-Command line utility for **Safe** contracts. Use it to manage your **Safe** easily from the command line.
+Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe account from the command line.
 
-Safe-CLI does not rely on _Safe{Core} API_, so it can be used in networks where Safe services are not available. If they are available,
-Safe-CLI can also interact with them in `tx-service` mode.
+It does not rely on Safe{Core} API and can also be used in networks where Safe services are unavailable. Learn more through the [documentation](https://docs.safe.global/advanced/cli-overview).
 
-## Using with docker
+## Using Docker
 
-If you have **Docker** installed on your system, you can just run:
+**Prerequisite:** Install [Docker Desktop](https://www.docker.com/products/docker-desktop/).
+
+Once Docker is installed on your system, run the following command to create new Safe accounts:
 
 ```bash
 docker run -it safeglobal/safe-cli safe-creator
 ```
 
-for creating Safes
-
-or
-
+You can also run the following command to run the Safe CLI with an existing Safe:
 ```bash
-docker run -it safeglobal/safe-cli safe-cli
+docker run -it safeglobal/safe-cli safe-cli <checksummed_safe_address> <ethereum_node_url>
 ```
 
-to run the actual **safe-cli**
-
-## Installing
+## Using Python PIP
 
-**Python >= 3.7** is required. **Python 3.10** is recommended.
+**Prerequisite:** [Python](https://www.python.org/downloads/) >= 3.7 (Python 3.10 is recommended).
```

Comment:
```suggestion
**Prerequisite:** [Python](https://www.python.org/downloads/) >= 3.9 (Python 3.12 is recommended).
```

Reply:
Thanks, updating this is the docs as well.